src/frontend/src/components/public/PIANavButton/index.tsx

```diff
@@ -1,42 +1,48 @@
 import { useLocation, useNavigate } from 'react-router-dom';
-import { PiaFormSideNavPages } from '../PIASideNav/pia-form-sideNav-pages';
+import { INavbarItem } from '../../common/Navbar/interfaces';
 import { INavButton } from './interface';
 
-const PIANavButton = ({ pia }: INavButton) => {
+const PIANavButton = ({ pages }: INavButton) => {
   const { pathname } = useLocation();
   const navigate = useNavigate();
+
+  const handleNavBtn = (direction: string) => {
+    const currentTabObj = pages.filter((page) => page.link === pathname);
+    let resultTab: INavbarItem[] = [];
+    if (direction === 'next') {
+      resultTab = pages.filter((page) => page.id === currentTabObj[0].id + 1);
```

Comment:
I do feel this is a super dangerous way to do it. 
For one, it 'id' is not generated and it is manually written. Something I am not happy with. 
Two, the order is based on how the object is structured in the array not the id. 

A better way is to get the index of the object on 'line 10'. and then check if (index>0) index-- and if (index <pages.length-1) index++

Reply:
let me re-write it. You are right, this implementation does not very robust